pom.xml

```diff
@@ -94,6 +94,25 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>com.rabbitmq</groupId>
+      <artifactId>http-client</artifactId>
+      <version>3.1.1.RELEASE</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
```

Comment:
why is Spring necessary for this test ?

Reply:
@vietj I use HOP client for RabbitMQ: https://github.com/rabbitmq/hop#maven

Reply:
can we avoid adding this dependency and use vertx client instead ?

Reply:
Ping